Sources/TaggedTime/TaggedTime.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+import Tagged
+
+public enum MillisecondsTag {}
+public typealias Milliseconds<A> = Tagged<MillisecondsTag, A>
+
+public enum SecondsTag {}
+public typealias Seconds<A> = Tagged<SecondsTag, A>
+
+extension Tagged where Tag == MillisecondsTag, RawValue: BinaryFloatingPoint {
+  public var seconds: Seconds<RawValue> {
+    return .init(rawValue: self.rawValue / 1000)
+  }
+
+  public var date: Date {
```

Comment:
right now this is constrained to both `BinaryFloatingPoint` and `BinaryInteger`. not sure if we can get more general than that.